sms-checker/README.md

```diff
@@ -2,7 +2,86 @@
 This folder contains the code for the SMS-checker application. This application can be triggered to query the database for any subscribers that are due an alert for a celestial body or aurora, this is done using the `psycopg2` library. These subscriber details are then used to send emails and SMS messages with the appropriate alerts using `boto3`.
 
 ## Setup
+1. Ensure that an SQL server RDS has been setup prior and is accessible
+    - Note: be sure to store the credentials for accessing this database safely.
+
+### To Run Locally:
+2. Setup a venv and install the requirements
+```bash
+python -m venv venv
+source venv/bin/activate
+pip install -r requirements.txt
+```
+3. Create a `.env` file with the following:
+    - DB_HOST - to access your RDS instance
+    - DB_USER - username for accessing RDS
+    - DB_PASSWORD - password for accessing RDS
+    - DB_NAME - the name of your database
+    - DB_PORT - port to access RDS (typically 5432 for postgres)
+    - AWS_ACCESS_KEY - your personal access key to AWS
+    - AWS_SECRET_KEY - your personal secret key to AWS
+    - REGION - the region of AWS that you will be using for this project
```

Comment:
Very informative, good to tell people exactly what the env variables require despite the descriptive names